File: HackerRank - Ulangi Sampai N/solver.py
# Ulangi Sampai N

n = int(input())

for x in range(1, n+1):
    if x % 5 == 0:
        continue
    elif(x % 10 == 3):
        print("Nice")
    elif(x >= 113):
        print("Error")
        break
    else:
        print(x)

File: HackerRank - Ulangi Sampai N/solver.cpp
#include <iostream>

using namespace std;

int main() {
    int n;
    cin >> n;

    for (int i = 1; i <= n; i++) {
        if (i % 5 == 0) {
            continue;
        } else if (i % 10 == 3) {
            cout << "Nice" << endl;
        } else if (i >= 113) {
            cout << "Error" << endl;
            break;
        } else {
            cout << i << endl;
        }
    }
}